.bazelrc

```diff
@@ -19,6 +19,9 @@ build:windows --cxxopt=/std:c++20 --cxxopt=/Zc:preprocessor --host_cxxopt=/std:c
 startup --windows_enable_symlinks
 common --enable_runfiles
 
+# with the above, we can avoid building python zips which is the default on windows as that's expensive
+build --nobuild_python_zip
```

Comment:
Ooooh amazing, I didn't know that option existed!

Reply:
Yeah, I found it mentioned in multiple places in the `rules_python` issues here and there. We'll need to set it internally as well.